src/main/webapp/WEB-INF/fenixedu-cms/changeMenu.jsp

```diff
@@ -344,8 +344,6 @@ ${portal.toolkit()}
 
     $("select[name=slugPage]").change(function(event) {
         $(event.target).closest('form').find('input[name=use].usepage').prop('checked', true);
-        var name = $(event.target).closest('form').find('input[name=name]');
-        name.val($(event.target).find(':selected').text());
```

Comment:
We need to do this right later. Probably we need to have JSON be placed in the select box to get the page later.